src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -417,8 +415,8 @@ private void androidSdkManager(String[] args) throws IOException, InterruptedExc
      * @throws InterruptedException in case anything goes wrong.
      */
     private void fetchFromSdkManager() throws IOException, InterruptedException {
-        Logger.logDebug("Downloading Android toolchain...");
+        Logger.logInfo("Downloading Android NDK and toolchain...");
```

Comment:
This step doesn't use `FileOps.downloadFile`, and there is no feedback. In my case it takes up to 5 minutes without feedback. 
Might be convenient to use something like `SubstrateDispatcher::startNativeCompileTimer`?

Reply:
I have added feedback by adding "." every second while the sdkmanager is running. This visual feedback should be enough for the user.

Reply:
no, a `.` is not enough. It is actually the most frustrating output one can get, unless you know how many dots are expected.

Reply:
What else would you suggest?

Reply:
If you don't know how far you are in the download (x%), adding the "this might take 5 minutes or longer" is better.